### ISSUE TITLE
Citizens won't fully stay/remove

### ISSUE THREAD

**Issue by backintime7** [CitizensDev/Citizens2#1335]
I try to remove or add a citizen.
Then, when I stop the server/reload citizens, the ones that I have added are removed, and the ones that are removed come back.

This is incredibly frustrating...

The problem persists even after I do /citizens save.

I don't want to delete this plugin! I can't make an RPG without Citizens

**Comment by Roy416** [comment]
Same

**Comment by Midnight9746** [comment]
You could try regenerating the config files made by the plugin. But make sure you save the file with the list of the npcs to your desktop to restore them.

Npcs should remove when deleted and stay when created.

You can also make sure you have the latest version for your server version. (Like no 1.12 server running 1.8 citizens).

Also check the config files if it continues and maybe even before to see if there's anything that could be affecting the npc.

And example would be, say if npcs skins won't auto update, the config might have:

SkinAutoUpdate: false

I'm late to this by 3 months ^-^

**Comment by mcmonkey4eva** [comment]
Is there any error in your log? If the NPC returns it may have been a saving failure, which would display an error.

**Comment by Janzski** [comment]
I'm having the same issue with no errors in console or debug file. I /citizens save'd and checked my saves.yml and it's totally empty though I have about six NPCs with text and skins and all--is that file supposed to be empty? I've restarted the server and the NPCs all remain, even the one I deleted (though they often randomly lose their skins).

**Comment by mcmonkey4eva** [comment]
if saves.yml is empty, and there are 6 NPCs after restart...

Well, you've got something wildly wrong there.

**Comment by Janzski** [comment]
Yeah I figured. I'm not sure how that would even happen. I'll try to delete the jar and re-download I guess?

**Comment by Janzski** [comment]
Okay, in case it helps someone else: deleting the jar seems to have worked. I suspect it may have something to do with the way I had the yml file open through my ftp program though because I closed that as well and I don't think I had before. So one of the two!
